libs/librrgraph/src/io/rr_graph_uxsdcxx_serializer.h

```diff
@@ -1727,6 +1729,17 @@ class RrGraphSerializer final : public uxsd::RrGraphBase<RrGraphContextTypes> {
      *     <xs:attribute name="tool_comment" type="xs:string" />
      *   </xs:complexType>
      */
+    inline void set_rr_graph_schema_file_id(unsigned long schema_file_id, void*& /*ctx*/) final {
+        // Only check if the schema file ID is not 0. If it is 0, it means capnproto is not enabled.
+        // Thus, we cannot check the schema file ID mismatch.
```

Comment:
I think this comment moment is incomplete. Don’t you also allow older rr graphs with no schema to come in? Might also want the to explain that we’re checking the file is compatible when can, but alllowing it to be read if we can’t determine its schema